rector.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use PhpCsFixer\Fixer\Import\NoUnusedImportsFixer;
+use Rector\Core\Configuration\Option;
+use Rector\Core\ValueObject\PhpVersion;
+use Rector\Set\ValueObject\LevelSetList;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
```

Comment:
Version 0.12.21 has been released meanwhile, introducing `Rector\Config\RectorConfig`. Should you wish to update 🥲

Reply:
Added, thats handy :)